modules/tinymce/src/core/main/ts/api/dom/Selection.ts

```diff
@@ -542,11 +544,22 @@ const EditorSelection = (dom: DOMUtils, win: Window, serializer: DomSerializer,
     controlSelection.destroy();
   };
 
+  /**
+   * Expands the range of the selection to contain the entire word when the selection is collapsed within the word
+   *
+   * @method expand
+   * @param {Object} options Optional options provided to the expansion. Defaults to { type: 'word' }
+   * @return {void}
```

Comment:
```suggestion
```
If something doesn't have a return we just don't specify the `@return` line normally.